storage/rocksdb/ha_rocksdb.h

```diff
@@ -149,6 +150,11 @@ const char *const BG_THREAD_NAME = "myrocks-bg";
 */
 const char *const INDEX_THREAD_NAME = "myrocks-index";
 
+/*
+  Name for the manual compaction thread.
+*/
+const char *const MANUAL_COMPACTION_THREAD_NAME = "myrocks-mc";
```

Comment:
Ideally
static const char constexpr MANUAL_COMPACTION_THREAD_NAME[] = "myrocks-mc";

Reply:
True, inherited from upstream but missed while fixing in port.